TerminalDocs/group-policy.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Group Policy
+description: Learn how to set Group Policy for Windows Terminal.
+author: chrnguyen
+ms.author: chrnguyen
+ms.date: 10/24/2024
+ms.topic: how-to 
+---
+
+# Group Policies
+
+Since Windows Terminal Preview 1.22, Windows Terminal is released on GitHub with Administrative Templates that allow you to configure Windows Terminal using Group Policies.
+
+## How to install
+
+### Download
+
+You can find the latest administrative templates (ADMX and ADML files) in the assets section of our newest Windows Terminal release on GitHub. The file is named `GroupPolicyTemplates-<Version>.zip`
+
+### Add the administrative template to an individual computer
+
+1. Copy the `WindowsTerminal.admx` file to your **Policy Definition** template folder. (Example: `C:\Windows\PolicyDefinitions`)
+2. Copy the `WindowsTerminal.adml` file to the matching language folder in your language folder in your Policy Definition folder. (Example: `C:\Windows\PolicyDefinitions\en-US`)
+
+> [!NOTE]
+> Copying the `WindowsTerminal.adml` file to the `en-US` folder will apply the policy even if you are not using an `en-US`-based machine.
+
+### Add the administrative template to Active Directory
+
+1. On a domain controller or workstation with RSAT, go to the **PolicyDefinition** folder (also known as the Central Store) on any domain controller for your domain. <br>
+For older versions of Windows Server, you might need to create the PolicyDefinition folder. For more information, see [How to create an manage the Central Store for Group Policy Administrative Templates in Windows](/troubleshoot/windows-client/group-policy/create-and-manage-central-store).
+2. Copy the `WindowsTerminal.admx` file to the **PolicyDefinition** folder. (Example: `%systemroot%\sysvol\domain\policies\PolicyDefinitions`)
+3. Copy the `WindowsTerminal.adml` file to the matching language folder in your language folder in your Policy Definition folder. Create the folder if it does not already exist (Example: `%systemroot%\sysvol\domain\policies\PolicyDefinitions\EN-US`)
```

Comment:
You can always use `en-us` as this is the fallback language. If no correct language exists `en-us` is used.

Or do you plan to provide translated admlnfor different languages?